iOS/PostEditor/MultilineTextView.swift

```diff
@@ -41,7 +41,7 @@ private struct UITextViewWrapper: UIViewRepresentable {
         uiView.font = fontMetrics.scaledFont(for: font)
 
         if uiView.window != nil && isEditing {
-            DispatchQueue.main.async {
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.55) {
```

Comment:
This delay the janky keyboard-presentation animation when navigating to a post.

| Before | After |
| :---: | :---: |
| <video src="https://user-images.githubusercontent.com/387655/210072805-66d5c83c-9870-4e76-bd58-f3e4926bf7aa.mov" /> | <video src="https://user-images.githubusercontent.com/387655/210072833-17a656de-7d0f-4753-8500-9e8084945e38.mov" /> |